pkg/osquery/table/launcher_info.go

```diff
@@ -63,9 +84,69 @@ func generateLauncherInfoTable(db *bbolt.DB) table.GenerateFunc {
 				"osquery_instance_id": osqueryInstance.InstanceId,
 				"fingerprint":         fingerprint,
 				"public_key":          publicKey,
+				// hardware encryption and signing keys refers to keys provided by either
+				// Apple's secure enclave or Linux / Windows TPM
+				"hardware_public_encryption_key":             hardwarePublicEncryptionKey,
+				"hardware_public_encryption_key_fingerprint": hardwarePublicEncryptionKeyFingerprint,
+				"hardware_public_signing_key":                hardwarePublicSigningKey,
+				"hardware_public_signing_key_fingerprint":    hardwarePublicSigningKeyFingerprint,
 			},
 		}
 
 		return results, nil
 	}
 }
+
+type keyer interface {
+	PublicSigningKey() (*rsa.PublicKey, error)
+	PublicEncryptionKey() (*rsa.PublicKey, error)
+}
```

Comment:
Thinking about this -- I think this should never change while launcher is running. So we can make them a constant somewhere